ens/src/ens-registry.ts

```diff
@@ -0,0 +1,175 @@
+import {
+  NewOwner as NewOwnerEvent,
+  NewResolver as NewResolverEvent,
+  NewTTL as NewTTLEvent,
+  Transfer as TransferEvent
+} from "../generated/EnsRegistry/EnsRegistry"
+import * as airstack from "../modules/airstack/domain-name";
+import {
+  log,
+} from "@graphprotocol/graph-ts";
+import { ETHEREUM_MAINNET_ID } from "../modules/airstack/domain-name/utils";
+import { TOKEN_ADDRESS_ENS } from "./utils";
+/**
+ * @dev this functions maps the NewOwner event to airstack trackDomainOwnerChangedTransaction
+ * @param event NewOwnerEvent from ENS Registry
+ */
+export function handleNewOwner(event: NewOwnerEvent): void {
+  log.info("handleNewOwner: owner {} node {} label {} txhash {}", [event.params.owner.toHexString(), event.params.node.toHexString(), event.params.label.toHexString(), event.transaction.hash.toHexString()]);
+  airstack.domain.trackDomainOwnerChangedTransaction(
+    event.block.number,
+    event.block.hash.toHexString(),
+    event.block.timestamp,
+    event.logIndex,
+    ETHEREUM_MAINNET_ID,
+    event.params.owner.toHexString(),
+    event.transaction.hash.toHexString(),
+    true,
+    event.params.node,
+    event.params.label,
+    TOKEN_ADDRESS_ENS,
+    false,
```

Comment:
part of module refactor

Reply:
I still feel this can be covered in this PR. You can place below code in calling code, outside module. 
```
if (fromOldRegistry && isMigratedMapping.isMigrated == true) {
      // this domain was migrated from the old registry, so we don't need to hanlde old registry event now
      return;
    }
   ```

Reply:
done